.github/composite_actions/get_platform_parameters/action.yml

```diff
@@ -5,62 +5,99 @@ inputs:
     required: true
     type: string
   xcode_version:
-    description: 'The version of Xcode. Valid values are 14.3 and 15.0'
-    required: true
+    description: "The version of Xcode. Available aliases are 'latest' and 'minimum'"
```

Comment:
Why not just add the new aliases but still support passing an explicit version? 

Because now `"latest`" is not really ***the*** latest (i.e. 15.0), so it's kinda misleading to call it like that.
If we have `latest = 15.0` set, but we explicitly pass `14.3` in our tests invocations, it's clear we need to investigate why we cannot upgrade.

So basically what I'm saying is keeping `14.0.1`,  `14.3` and `15.0` as allowed value, but also supporting `latest` which we consider to be `15.0` and `minimum` which we consider to be `14.0.1`.

Then you can just use `latest` in tests that support it, but set `14.3` in the ones that don't.

Reply:
It is possible to pass a specific version. The alias available is only `latest` and `minimum`. You can still pass `15.1` as `xcode_version` input. However, you will also need to specify the `destination` for it.

Reply:
Aah I see, thanks for clarifying that :) 

If setting an explicit destination is indeed an "edge" case (i.e. when testing new beta versions, or for a reduced number of tests), then that's fine. But if we have to write the whole `destination` in many files, then this action becomes irrelevant and exactly what it tried to replace :P

tl;dr: this is fine as long as we don't have to set an explicit xcode version in more than 1 workflow, otherwise we should add the mapped destination for that xcode version and save ourselves one long and messy input.

Reply:
yeah, if that's the case, we might need to consider adding another alias `beta` and the destination mapping in the script.